llvm/lib/Target/DirectX/DXIL.td

```diff
@@ -738,6 +739,16 @@ def UMin : DXILOp<40, binary> {
   let attributes = [Attributes<DXIL1_0, [ReadNone]>];
 }
 
+def UAddc : DXILOp<44, binaryWithCarryOrBorrow > {
+  let Doc = "Unsigned 32-bit integer arithmetic add with carry. uaddc(a,b) = (a+b, a+b overflowed ? 1 : 0)";
```

Comment:
We should be consistent and match the docs in [DXIL.rst](https://github.com/microsoft/DirectXShaderCompiler/blob/main/docs/DXIL.rst#operations-via-external-functions) for these (even though in this case that wording is a little awkward).
```suggestion
  let Doc = "unsigned add of 32-bit operand with the carry";
```